external/kafka-assembly/pom.xml

```diff
@@ -66,13 +66,16 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey</groupId>
-      <artifactId>jersey-server</artifactId>
-      <scope>provided</scope>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-client</artifactId>
```

Comment:
I'd like to know more precisely why the dependency was marked as provided in the first place. Any ideas?